syncd/VendorSai.cpp

```diff
@@ -24,6 +24,40 @@ VendorSai::VendorSai()
     m_apiInitialized = false;
 
     memset(&m_apis, 0, sizeof(m_apis));
+
+    sai_global_apis_t ga =
```

Comment:
from now on, we keep sai global api pointers in a metadata generated structs, this will be handy when we will want to load another vendor libsai.so into memory